adventure/Ye_olde_store_directory.yaml

```
@@ -3,7 +3,7 @@
 Toadstool Tavern: 11
 Swords'r'us: 14
 Percival's Potions: 9
-Bitchin' Witch Supply Co.: 13
+Bitchin' Witchin' Supply Co.: 13
 Got Potions?: 15
 Scrolls and Spells: 17
-Pricey Potions: 24
\ No newline at end of file
+Pricey Potions: 24
```

I am currently performing this code review using the "GitHub Pull Requests and Issues" extension in VSCode. I notice that the GitHub interface complains about this line here, suggesting "No newline at end of file". Now I usually do have newlines at the end of Python files, but haven't done so for YAML files. Maybe we should put a newline here as well.

```suggestion
Pricey Potions: 24

```